projects/scannet_offline_eval/demo/proto_robot/robodata_server.py

```diff
@@ -53,7 +92,7 @@ def Chat(self, request, context):
         prompt = "Task: You are a chatbot exploring a home."
```

Comment:
Same deal -- since this is duplicated we probably should have one place that implements this prompt